awx/main/migrations/0064_v350_inventory_compat_mode.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2019-02-28 16:22
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0063_v350_org_host_limits'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='inventorysource',
+            name='compatibility_mode',
```

Comment:
I know this is late, but I don't love this name. Even though we have a description and it gives some context it'd be nice if you could look at  this field name and know that it has something to do with emulating old host[vars] behavior.

Reply:
If I had to rank the importance of the differences that using compatibility mode has:

1. group names are allowed to use "unsafe" characters
2. additional hostvars, what you mentioned
3. host names will (sometimes, when we can) use the naming that the script used
4. group_by behavior where it is left blank, to better emulate how plugins behave

there's probably more after that.